common/src/cellular_at_core.c

```diff
@@ -31,17 +31,21 @@
 #include <string.h>
 #include <ctype.h>
 
-#include "cellular_config.h"
+#ifndef CELLULAR_DO_NOT_USE_CUSTOM_CONFIG
```

Comment:
Can you describe the scenario about this macro more?

Reply:
cellular_config.h needs to be provided by users or it will be built failed. In our case, we only want to build library without any user's modification.

Reply:
It is reasonable that we used default config when doing the test.
Thanks for the update to add this config in other files in cellular library.